Makefile

```diff
@@ -12,80 +15,86 @@ GREEN=`tput setaf 2`
 RESET=`tput sgr0`
 YELLOW=`tput setaf 3`
 
-all: .installed.cfg
+PLONE5=5.2.7
+PLONE6=6.0.0a4
```

Comment:
Suggestion: use 5.2-latest and 6.0-latest.

Reply:
@mauritsvanrees that's a common anti-pattern in the Plone community IMHO. If your CI system relies on an external resource that is subject to change you can not trust your CI on anything. For any failure you have to ask yourself "is that because of my change or because of an external change?". This is one of the (if not THE) basic rules of CI. If you ignore that rule you can forget about a reliable CI.

Reply:
The danger of pinning it to a specific version, is that no one updates the version, and a year from now you are still testing against alpha 4. See `plone.restapi` which is tested against [Plone 5.2.6](https://github.com/plone/plone.restapi/blob/master/plone-5.2.x.cfg#L3) instead of 5.2.7. 

But yes, as you know we had problems testing `plone.restapi` on Plone 6 because it checked out master of `Products.CMFPlone`. A specific commit would have been better.

In the current case, `plone.volto` is tested in `buildout.coredev` with latest Plone under development. So it is okay to test it with exact version pins here. Still, I would be in favour of using `6.0-latest`. That should not cause problems, exactly because it is already tested in coredev.

But: just a suggestion.